src/timesignature.ts

```diff
@@ -0,0 +1,137 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+//
+// ## Description
+// Implements time signatures glyphs for staffs
+// See tables.js for the internal time signatures
+// representation
+
+import { Vex } from './vex';
```

Comment:
Tough to review this file, any way to make it show up as a diff in here?

Reply:
@0xfe You got the diff above :)